src/JBFundAccessLimits.sol

```diff
@@ -11,7 +9,7 @@ import {JBFundAccessLimitGroup} from "./structs/JBFundAccessLimitGroup.sol";
 
 /// @notice Stores and manages terminal fund access limits for each project.
 /// @dev See the `JBFundAccessLimitGroup` struct to learn about payout limits and surplus allowances.
-contract JBFundAccessLimits is JBControlled, ERC165, IJBFundAccessLimits {
+contract JBFundAccessLimits is JBControlled, IJBFundAccessLimits {
```

Comment:
same (is ierc165)?

Reply:
if we want to do this, we should also do so for Splits, Rulesets, and Tokens imo.